tests/utils/sandbox-child.mjs

```diff
@@ -1,109 +1,53 @@
-import { Buffer } from 'node:buffer'
-import { join } from 'node:path'
+// @ts-check
 
-import { execute, getLogger } from 'lambda-local'
-
-const SERVER_HANDLER_NAME = '___netlify-server-handler'
-const BLOB_TOKEN = 'secret-token'
+import { getLogger } from 'lambda-local'
+import { loadFunction } from './lambda-helpers.mjs'
 
 getLogger().level = 'alert'
 
-const createBlobContext = (ctx) =>
-  Buffer.from(
-    JSON.stringify({
-      edgeURL: `http://${ctx.blobStoreHost}`,
-      uncachedEdgeURL: `http://${ctx.blobStoreHost}`,
-      token: BLOB_TOKEN,
-      siteID: ctx.siteID,
-      deployID: ctx.deployID,
-      primaryRegion: 'us-test-1',
-    }),
-  ).toString('base64')
-
-function streamToBuffer(stream) {
-  const chunks = []
-
-  return new Promise((resolve, reject) => {
-    stream.on('data', (chunk) => chunks.push(Buffer.from(chunk)))
-    stream.on('error', (err) => reject(err))
-    stream.on('end', () => resolve(Buffer.concat(chunks)))
-  })
-}
-
-process.on('message', async (msg) => {
-  if (msg?.action === 'exit') {
-    process.exit(0)
-  } else if (msg?.action === 'invokeFunction') {
-    try {
+/**
+ * @type {import('./lambda-helpers.mjs').InvokeFunction | undefined}
+ */
+let invokeFunctionImpl
+
+process.on(
+  'message',
+  /**
+   * @param {any} msg
+   */
+  async (msg) => {
```

Comment:
This wasn't typed before, but adding explicit note was needed after adding `// @ts-check` to get at least partial type checking here and is this is not used directly in tests I didn't want to spend too much time on making details of sandbox fully typed